web/src/peer_data.ts

```diff
@@ -158,3 +158,22 @@ export function is_user_subscribed(stream_id: number, user_id: number): boolean
     const subscribers = get_user_set(stream_id);
     return subscribers.has(user_id);
 }
+
+export function count_guest_visible_users(stream_ids: number[]): number {
```

Comment:
A better name for this function would be `get_unique_subscriber_count_for_streams` or probably something else but not something mentioning about guests.